data.sql

```diff
@@ -4,3 +4,10 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', 'Nov 15 2018', 2, true, 8);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (3, 'Pikachu', 'Jan 7 2021', 1, false, 15.04);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (4, 'Devimon', 'May 12 2017', 5, true, 11);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (5, 'Charmander', 'Feb 8 2020', 0, false, 11);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (6, 'Plantmon', 'Nov 15 2021', 2, true, 5.7);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (7, 'Squirtle', 'Apr 2 1993', 3, false, 12.13);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (8, 'Angemon', 'Jun 12 2005', 1, true, 45);
```

Comment:
- Please note that the `weight_kg` of lines 7 -10 are `-11`, `-5.7`, `-12.13`, and `-45` respectively. I guess you simply missed the negative sign, no worries it happens to the best of us 😜. Kindly add them to get the right result for your queries 👍🏽